docroot/themes/contrib/civictheme/civictheme_library/components/00-base/typography/typography.twig

```diff
@@ -0,0 +1,20 @@
+{#
```

Comment:
we do not have a Typography component. In Storybook  we are just showing how typography will look like without any themeing.

Please remove this.